MediaBrowser.Common/Plugins/IPluginRegistrar.cs

```diff
@@ -5,7 +5,7 @@ namespace MediaBrowser.Common.Plugins
     /// <summary>
     /// Defines the <see cref="IPluginRegistrar" />.
     /// </summary>
-    public interface IPluginRegistrar
+    public interface IPluginServiceRegistrator
```

Comment:
btw, you manually have to rename the file and fix the references

Reply:
thanks twas late here. Just renamed.